ansible/Makefile

```diff
@@ -46,26 +46,40 @@ osp_tests_run: hosts local-defaults.yaml
 	-i hosts -e @local-defaults.yaml \
   20_tempest_ocp.yaml \
 
-podified-ctlplane: hosts ctlplane
-ctlplane: local-defaults.yaml
-	# NOTE: requires 'make olm'
+# Services which depend on olm
+olm_services=mariadb keystone glance
+
+# Services which don't depend on olm
+yaml_services=rabbitmq ovn neutron-api nova-api nova-cell placement
+
+# Run a specific install playbook without dependencies
+install_%: install_%.yaml hosts local-defaults.yaml
 	ANSIBLE_FORCE_COLOR=true ansible-playbook \
-	-i hosts -e @local-defaults.yaml \
-	install_namespace.yaml \
```

Comment:
I personally kind of like the hackability of what you are removing here for now. I would also point out that all of this goes away soon anyway once we have a ctlplane operator so why not just wait on that to run things in parallel?

Reply:
I'd argue this patch improves hackability. The main reason I've kept it separate is I'm not 100% sure about the trade off in complexity. It's not very complex, but it's certainly more complex that just a list of targets.

The explicit ordering isn't just about parallelism, although that's what I was interested in when I wrote it (incidentally: as a toy. It took me a bit over 30 minutes.) I've discovered a few bugs which would have hit us in CI, and one hard deployment ordering dependency that I'd previously believed was a soft dependency.

Something I have found myself missing when working on other branches, though, is the syntactic sugar of the install_<foo> target. It's much nicer to type 'make install_ovn' than 'ansible-playbook -i hosts -e @local-default.yaml install_ovn.yaml'. I might pull that out as a separate PR.